sql/npc_list.sql

```diff
@@ -17028,9 +17028,9 @@ INSERT INTO `npc_list` VALUES (17346614,'Armoury_Crate','Armoury Crate',46,83.00
 INSERT INTO `npc_list` VALUES (17346624,'Armoury_Crate','Armoury Crate',46,-396.159,94.211,-64.397,6,40,40,0,8,0,6,1155,0x0000C10300000000000000000000000000000000,0,NULL,1); -- dropping_like_flies
 INSERT INTO `npc_list` VALUES (17346634,'Armoury_Crate','Armoury Crate',46,-156.000,-25.000,115.000,6,40,40,0,8,0,6,1155,0x0000C10300000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17346644,'Armoury_Crate','Armoury Crate',46,83.000,-145.000,295.000,6,40,40,0,8,0,6,1155,0x0000C10300000000000000000000000000000000,0,NULL,1);
-INSERT INTO `npc_list` VALUES (17346653,'Armoury_Crate','Armoury Crate',46,-396.159,94.211,-64.397,6,40,40,0,8,0,6,1155,0x0000C10300000000000000000000000000000000,0,NULL,1); -- tails_of_woe
```

Comment:
I would say place back the comment showing that this armoury crate is for tails of woe but that is my personal opinion ^^;;